src/icons/__generated/Circle2.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/display-name */
```

Comment:
can we rename to MapCircle

Reply:
these names came from figma and I like to match the figma names, we could rename them there  as well